README.md

```diff
@@ -51,6 +51,12 @@ into the console:
 $ ./go test
 ```
 
+Once that completes successfully, use the same script to run the Continuous Integration suite:
```

Comment:
`./go ci_build` incorporates the `test` command. Perhaps this should be: `To run the tests and then attempt to build the gem, run the continuous integration command:`.